src/terraform/devcontainer-feature.json

```
@@ -1,6 +1,6 @@
 {
     "id": "terraform",
-    "version": "1.2.0",
+    "version": "1.2.1",
```

```suggestion
    "version": "1.3.0",
```

Can we bump the patch version as this PR adds a new option?